src/sagas/authSaga.js

```diff
@@ -0,0 +1,27 @@
+import { put, call, takeLatest } from "redux-saga/effects";
+import { actionTypes } from "../actions/types";
+import authService from "../service/authService";
+import {
+  loginSuccess,
+  loginError,
+  getErrors,
+} from "../actions/authActions";
+
+function* loginSaga(action) {
+  try {
+    const { email, password } = action.payload;
+    const response = yield call(authService.login, { email, password });
```

Comment:
you can send action.payload because it is {email, password}. You don't need to destruct in line 12 and send as object here